### DISCUSSION TITLE
Private or Password protected Garden

### DISCUSSION THREAD

**Discussion by ArneNostitz** [oleeskild/obsidian-digital-garden#166] [General]
Hi, this is a really awesome thing to have a knowledge base for my team. I was wondering if there is a way to protect this with a login? 

thanks

**Comment by ArneNostitz** [comment]
(also having the github repo private would be key) thanks

**Comment by oleeskild** [comment]
You can make your repo private, that would not break anything. The password protection is a bit harder. Currently what you can do is using the [password protection](https://docs.netlify.com/visitor-access/password-protection/) feature from Netlify. Unfortunately this means you'll have to upgrade to a paid plan.

**Comment by CubityFirst** [comment]
Using Cloudflare Zero Trust & Cloudflare Access is a bit of effort, but not that bad to set up & let's you OAuth with services like Google etc.
